### ISSUE TITLE
Tab headers in "Series details" dialog  do not meet minimum color contrast ratio

### ISSUE THREAD

**Issue by luniki** [opencast/opencast-admin-interface#597]
The text in the Tab headers in "Series details" dialog do not meet the minimum color contrast ratio.

The ratio should be at least 4.5:1.

This has a serious impact on people with low vision.

![Screenshot 2024-06-05 at 09-58-05 Opencast](https://github.com/opencast/opencast-admin-interface/assets/2993/56a27dcd-5791-41b7-b21c-ce18d4816102)
![Screenshot 2024-06-05 at 09-57-53 Opencast](https://github.com/opencast/opencast-admin-interface/assets/2993/b8df8726-1a2f-45a4-9abc-6a88e8fe4f1b)
